tests/foreman/virtwho/api/test_esx.py

```diff
@@ -177,7 +177,7 @@ def test_positive_debug_option(self, default_org, form_data, virtwho_config, tar
 
         /foreman_virt_who_configure/api/v2/configs/:id"
 
-        :id: be395108-3944-4a04-bee4-6bac3fa03a19
+        :id: 24bf4c5b-cc7a-40ac-9079-e59c73984081
```

Comment:
@yanpliu Could you please revert this change for the test case ids? it seems irrelevant and resolve merge conflict